### PR TITLE
Update WebContext.php

### PR DESCRIPTION
**Pull request by gonzalovilaseca**
IMO this step should check the current url, not visiting a product page.

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
:+1:

File: src/Sylius/Bundle/WebBundle/Behat/WebContext.php [hunk 1]
```diff
@@ -468,9 +468,9 @@ public function iAmOnTheOrderPage($action, $number)
      */
     public function iShouldBeOnTheOrderPage($action, $number)
     {
-        $this->iAmOnTheOrderPage($action, $number);
+        $product = $this->findOneBy('product', array('name' => $name));
```
Comment (pamil) [inline src/Sylius/Bundle/WebBundle/Behat/WebContext.php]: There's no name in this method, I think it's the change for `iShouldBeOnTheProductPage($name)` at line 447.

Comment (gonzalovilaseca) [inline src/Sylius/Bundle/WebBundle/Behat/WebContext.php]: I messed with the online editor...will update PR..

**Comment by gonzalovilaseca** [comment]
Updated PR adding also order page.

File: src/Sylius/Bundle/WebBundle/Behat/WebContext.php [hunk 2]
```diff
@@ -468,9 +468,9 @@ public function iAmOnTheOrderPage($action, $number)
      */
     public function iShouldBeOnTheOrderPage($action, $number)
     {
-        $this->iAmOnTheOrderPage($action, $number);
+        $order = $this->findOneBy('order', array('number' => $number));
```
Comment (pamil) [inline src/Sylius/Bundle/WebBundle/Behat/WebContext.php]: I guess there's no point in getting order with given number from database and then using only order number :)

Comment (gonzalovilaseca) [inline src/Sylius/Bundle/WebBundle/Behat/WebContext.php]: :-)

**Comment by pamil** [comment]
Looks good for me :+1: :)

**Comment by pjedrzejewski** [comment]
Thank you Gonzalo! :)

**Pull request merged**